app/assets/stylesheets/_header.scss

```diff
@@ -7,5 +7,15 @@
     display: block;
     margin-top: 5px;
     font-weight: normal;
+    padding: 0;
+
+    & li {
```

Comment:
You shouldn't need the `&` here should you? It is implicit when you are nesting selectors like this.